tests/page-objects/navigation.ts

```diff
@@ -16,9 +16,7 @@ export class Navigation {
         observer.disconnect();
       });
       observer.observe(element);
-      // Firefox doesn't call IntersectionObserver callback unless
-      // there are rafs.
-      requestAnimationFrame(() => {});
+      requestAnimationFrame(() => undefined);
```

Comment:
I think it might be worth leaving that comment in, as it gives good context as to why that call is present.